.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,28 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '31 18 * * 4'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      with:
+        languages: 'javascript'
+        queries: security-and-quality
```

Comment:
I've added `security-and-quality` here so it'll run more than just security queries and will find general programming errors as well as maintainability and reliability issues. You can read more at https://docs.github.com/en/code-security/secure-coding/configuring-code-scanning#running-additional-queries